jib-cli/src/test/java/com/google/cloud/tools/jib/cli/jar/JarFilesTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.cli.jar;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.tools.jib.api.InvalidImageReferenceException;
+import com.google.cloud.tools.jib.api.JibContainerBuilder;
+import com.google.cloud.tools.jib.api.buildplan.AbsoluteUnixPath;
+import com.google.cloud.tools.jib.api.buildplan.ContainerBuildPlan;
+import com.google.cloud.tools.jib.api.buildplan.FileEntriesLayer;
+import com.google.cloud.tools.jib.api.buildplan.ImageFormat;
+import com.google.cloud.tools.jib.api.buildplan.Platform;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.io.Resources;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.Instant;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class JarFilesTest {
+  private static final String SIMPLE_STANDARD_JAR = "jar/standard/basicStandardJar.jar";
+
+  @Rule public final TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+  @Test
+  public void testToJibContainerBuilder_basicInfo()
+      throws IOException, URISyntaxException, InvalidImageReferenceException {
+    Path standardJar = Paths.get(Resources.getResource(SIMPLE_STANDARD_JAR).toURI());
+    Path destDir = temporaryFolder.getRoot().toPath();
+    JibContainerBuilder containerBuilder = JarFiles.toJibContainerBuilder(standardJar, destDir);
+    ContainerBuildPlan buildPlan = containerBuilder.toContainerBuildPlan();
+
+    assertThat(buildPlan.getBaseImage()).isEqualTo("gcr.io/distroless/java");
+    assertThat(buildPlan.getPlatforms()).isEqualTo(ImmutableSet.of(new Platform("amd64", "linux")));
+    assertThat(buildPlan.getCreationTime()).isEqualTo(Instant.EPOCH);
+    assertThat(buildPlan.getFormat()).isEqualTo(ImageFormat.Docker);
+    assertThat(buildPlan.getEnvironment()).isEmpty();
+    assertThat(buildPlan.getLabels()).isEmpty();
+    assertThat(buildPlan.getVolumes()).isEmpty();
+    assertThat(buildPlan.getExposedPorts()).isEmpty();
+    assertThat(buildPlan.getUser()).isNull();
+    assertThat(buildPlan.getWorkingDirectory()).isNull();
```

Comment:
Food for thought: these are certainly interesting things, and I guess the user should be allowed to configure them. Then we get back to the same question we had for the usual Jib build command. Originally, everything was through command-line options and we didn't have a "build file." This makes me think, maybe, the `jib jar` subcommand should also support a "build file." Then the `layers` field should probably be interpreted in a different way. I don't know.

Reply:
Yeah, that's definitely an interesting option to consider. The advantage of this would be that the user would be allowed to configure these params without having write a really long command. On the other hand (maybe it is too early to tell) could this potentially blur the line between the `jib build` command and the `jib jar` command or cause some confusion? For example, if I need the build file anyways, I might be better off using the `jib build` command instead? --> I could be missing some viable use cases here. 

This also makes me think: Since our general idea for the jar command is to require minimal user input, is it worth picking some optimal defaults (like workingDirectory or user,  for example)?

Reply:
> if I need the build file anyways, I might be better off using the jib build command instead?

`jib jar` still does way more things, like optimally creating multi layers (especially having a more tailored logic for Spring Boot), setting an entrypoint with right classpath, etc. And I envision that it will provide a nice getting-started experience and cover simple use cases without having to write a build file. At the same time, it can also support a build file, so that people who need advanced and complex configurations can consider that as an option for convenience.

And definitely `jib jar` is designed to be Java and JAR-specific, so it makes sense to set optimal defaults for many configurations. I actually think we aim for that. We can be highly opinionated on that. Actually, we are already so by creating layers as we think is better.